packages/db/schemas/registrations.ts

```diff
@@ -15,6 +15,7 @@ export const registrations = pgTable(
       .references(() => happenings.slug),
     status: registrationStatusEnum("status").notNull().default("waiting"),
     unregisterReason: text("unregister_reason"),
+    //updateReason: text("update_reason"),
```

Comment:
```suggestion

```

fjern denne linjen.

Reply:
tenkte å høre på møtet på onsdag som va om det skal være en egen reason for update registration, sånn det e no blir bare reason endret

Reply:
Jeg kan fikse det i #846 .